src/Skjema/Sprak.elm

```diff
@@ -0,0 +1,48 @@
+module Skjema.Sprak exposing (SpråkSkjema(..), SpråkSkjemaInfo, encode, init)
+
+import Cv.Spraakferdighet as Spraakferdighet exposing (Spraakferdighet)
+import Json.Encode
+
+
+type SpråkSkjema
+    = SpråkSkjema SpråkSkjemaInfo
+
+
+type alias SpråkSkjemaInfo =
+    { sprak : String
+    , muntlig : String
+    , skriftlig : String
```

Comment:
Hvorfor er dette Strings? Burde det ikke være en kode-type for språk, og "hvor-godt-kan-jeg-dette" type for muntlig og skriftlig?

Reply:
Jeg skjønner egentlig ikke hvorfor du har denne filen i det hele tatt, når du har `EnkeltSpråk`. Kan du ikke bare bruke den?